core/src/main/scala/cromwell/core/WorkflowOptions.scala

```diff
@@ -42,10 +42,26 @@ import scala.util.{Failure, Success, Try}
  */
 
 object WorkflowOptions {
+
+  sealed trait WorkflowOption { def name: String }
```

Comment:
There's an argument that `sealed abstract class` is better than `sealed trait` regardless. I've not yet decided if I agree w/ that and certainly don't care enough to comment in a PR but since there's already a suggestion for abstract class, there ya go